src/utils/components/Consoles/components/vnc-console/VncConsole.tsx

```diff
@@ -201,9 +210,13 @@ export const VncConsole: FC<VncConsoleProps> = ({
               text: 'Ctrl + Alt + 2',
             },
           ]}
+          onInjectTextFromClipboard={(e: MouseEvent<HTMLButtonElement>) => {
+            e.currentTarget.blur();
+            e.preventDefault();
+            rfb?.sendPasteCMD();
```

Comment:
Is there any way to focus on the console section too?

Reply:
yes. rfb.focus()
but I didn't like it as it makes the screen jump down